pkg/agent/flowexporter/priorityqueue/priorityqueue.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the “License”);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an “AS IS” BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package priorityqueue
+
+import (
+	"container/heap"
+	"time"
+
+	"antrea.io/antrea/pkg/agent/flowexporter"
+)
+
+var minExpiryTime = 100 * time.Millisecond
```

Comment:
do you change this value in unit tests? If not, shouldn't it be `const`?

Reply:
also maybe add a comment for this value

Reply:
yes, it should be `const`.
comment added.